docs/workforce-management-framework/topics/ref-mobile.adoc

```diff
@@ -1,25 +1,20 @@
 [id='{context}-ref-mobile']
-= {PROJECT_NAME} Mobile Application
+= {PROJECT_NAME} Mobile application
```

Comment:
This was changed by John to capitals. 
Did you changed that back or it was some rebase problem?